packages/slider/src/HandleController.ts

```diff
@@ -522,12 +548,17 @@ export class HandleController {
                     aria-label=${ifDefined(model.ariaLabel)}
                     aria-labelledby=${ariaLabelledBy}
                     aria-valuetext=${this.formattedValueForHandle(model)}
+                    aria-describedby="sliderDesc"
                     @change=${this.onInputChange}
                     @focus=${this.onInputFocus}
                     @blur=${this.onInputBlur}
                     @keydown=${this.onInputKeydown}
                     .model=${model}
                 />
+                <p id="sliderDesc" class="visually-hidden">
+                    Press escape or double click to reset the slider to its
```

Comment:
Please add an entry for this content to https://github.com/adobe/spectrum-web-components/issues/1975